src/Fixer/Operator/BinaryOperatorSpacesFixer.php

```diff
@@ -111,4 +180,21 @@ private function isDeclareStatement(Tokens $tokens, $index)
 
         return false;
     }
+
+    private function runHelperFixers(\SplFileInfo $file, Tokens $tokens)
+    {
+        $helpersToRunCount = count($this->alignFixerHelpers);
+        if (0 === $helpersToRunCount) {
+            return;
+        }
```

Comment:
all 4 lines could be removed, that early return doesn't provide any good here, the same will happen if array is empty